Dependencies/napi/napi-common/Include/napi/env.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
```

Comment:
Like we discussed in chat, I'm not sure we should be sharing headers between napi-jsi and napi-direct since they are logically separate things that probably shouldn't be in the same repo.